tests/dict_import_export_tests.py

```diff
@@ -66,8 +66,12 @@ def tearDownClass(cls):
         cls.delete_imports()
 
     def create_table(
-        self, name, schema="", id=0, cols_names=[], cols_uuids=None, metric_names=[]
+        self, name, schema="", id=0, cols_names=None, cols_uuids=None, metric_names=None
```

Comment:
Pylint is already capable of capture this error. We just didn't enable it for tests.

Reply:
In addition to the pylint rules, DeepSource also detects some performance issues, bug-risks and security vulnerabilities. Also, you can Autofix them with just one click. DeepSource will flag only those changes which occur in a particular patch/PR, which will make the code review process faster.
Just like the pylint config, you can selectively choose which issues to ignore in DeepSource too (test files in this case).
Feel free to reach out or check out the [docs](https://deepsource.io/docs/) if you have any questions :)